manifests/shosts_equiv.pp

```diff
@@ -4,17 +4,23 @@
 # used in a standalone fashion (with proper Hiera data) but
 # intended to be used indirectly by including the target class.
 #
+# @param shosts_equiv_file
+#   Path to shosts.equiv file. Set to undef/null if no shosts.equiv
+#   file should be managed.
+#
 # @example
 #   include profile_hostbased_ssh::shosts_equiv
 class profile_hostbased_ssh::shosts_equiv (
-  String $shosts_equiv_file,
+  Optional[String] $shosts_equiv_file,
```

Comment:
Again similar question/suggestion...

I'd have the conditional also be based on if `$profile_hostbased_ssh::known_hosts::hosts_data` is not empty/undefined`.

My brain just wouldn't default to thinking about changing the value of `$shosts_equiv_file`. But would rather assume that if I didn't supply any `$profile_hostbased_ssh::known_hosts::hosts_data` data it would do the right thing.

Reply:
Ditto.